src/App.js

```diff
@@ -58,6 +58,16 @@ function App() {
       setTodos([...todos, newTodo]);
     }
   };
+  const editTodo = (id) => {
+    let text = window.prompt('Edit Todo', '');
```

Comment:
I'll merge this for now but it is not good to use merge. Maybe you can create a modal

Reply:
I will work on that!

Reply:
Make a Pull so that you keep up with the latest code. Thanks for the edit functionality